tests/units/Glpi/Kernel.php

```diff
@@ -125,6 +125,8 @@ public function getDb() {
 
    /**
     * Test compilation of services configured in a plugin.
+    *
+    * @ignore
```

Comment:
What is the purpose?

Reply:
Ignoring the test, without having to remove the method.

Reply:
ok :)